src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+@import "../utils/_variables";
+
+@import "../utils/mixins";
+
+@import "../utils/normalize";
```

Yes, import these in main file. Also, no need for empty lines.
```suggestion
@import "../utils/_variables";
@import "../utils/mixins";
@import "../utils/normalize";
```